nipype/utils/filemanip.py

```diff
@@ -425,6 +426,18 @@ def copyfiles(filelist, dest, copy=False, create_new=False):
             newfiles.insert(i, destfile)
     return newfiles
 
+def mkdirp(folder):
+    """
+    Equivalent to bash's mkdir -p
+    """
+    if not os.path.exists(folder):
```

Comment:
You might want to use `os.path.isdir`, which will throw an `EEXIST` if a non-directory exists.